### PR TITLE
Feature/hdf 97 update main css stylesheet

### PR DESCRIPTION
**Pull request by tracyarciaga**
## Description of Changes

- Update colors and fonts in theme.
- Change CSS for BalancerPool,  PoolSummary and Header components.
- Add new images.

[Link to Jira Ticket -  HDF 97](https://halodao.atlassian.net/browse/HDF-97?atlOrigin=eyJpIjoiYzQwZGJhY2NmNDE5NDdmNDkzMTcyMzlhN2I4N2U1NjEiLCJwIjoiaiJ9)
[Link to Jira Ticket  -  HDF 98](https://halodao.atlassian.net/browse/HDF-98)
[Link to Jira Ticket  -  HDF 99](https://halodao.atlassian.net/browse/HDF-99)
[Link to Jira Ticket  -  HDF 100](https://halodao.atlassian.net/browse/HDF-100)

### Developer Checklist:

* [x] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/HDF-XXX-description`
* [x] All files have appropriate file headers and documentation
* [x] I have written new tests for your core changes, as applicable
* [ ] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

**Comment by schystz** [comment]
Hi @tracyarciaga , can you please rebase this branch to develop so there are less files/changes to review on Files changed tab? I believe that should also solve the code conflicts shown below.

**Comment by tracyarciaga** [comment]
@schystz rebase done.

**Comment by schystz** [comment]
> @schystz rebase done.

Looks like rebase failed @tracyarciaga  -

Screen Shot 2021-04-12 at 3 36 55 PM

**Comment by tracyarciaga** [comment]
> > @schystz rebase done.
> 
> Looks like rebase failed @tracyarciaga -
> 
> Screen Shot 2021-04-12 at 3 36 55 PM

hmmm
.. I don't understand why, also there are no conflicts already.  The file 
changed tab contains only the actual files that have been changed too.
@schystz

**Comment by schystz** [comment]
Yeah it looks like this is because of the previous merge in `develop` - the commits where directly pushed to develop branch for some reason instead of the branch (`features/hdf-8-view-stats-per-pool`) merged to `develop`. But it looks like the PR is all good now, I can continue to review. Thanks @tracyarciaga ! :)

**Comment by tracyarciaga** [comment]
> Yeah it looks like this is because of the previous merge in `develop` - the commits where directly pushed to develop branch for some reason instead of the branch (`features/hdf-8-view-stats-per-pool`) merged to `develop`. But it looks like the PR is all good now, I can continue to review. Thanks @tracyarciaga ! :)

Thank you!  I also fixed the ESlint warnings.

File: public/locales/en.json [hunk 1]
```diff
@@ -87,5 +87,10 @@
   "forAtLeast": "for at least ",
   "brokenToken": "The selected token is not compatible with Uniswap V1. Adding liquidity will result in locked funds.",
   "toleranceExplanation": "Lowering this limit decreases your risk of frontrunning. However, this makes more likely that your transaction will fail due to normal price movements.",
-  "tokenSearchPlaceholder": "Search name or paste address"
+  "tokenSearchPlaceholder": "Search name or paste address",
+  "farm": "Farm",
+  "stake": "Stake",
+  "poolSummaryStakeable": "My Stakeable Value",
+  "poolSummaryStaked": "My Staked Value",
+  "poolSummaryHaloEarned": "My HALO Earned"
```
Comment (schystz) [inline public/locales/en.json]: Nice catch @tracyarciaga , I forgot this codebase uses localization so we should take advantage of it too

File: src/pages/BalancerPool/index.tsx [hunk 2]
```diff
@@ -32,33 +61,58 @@ const BalancerPool = () => {
...
+            <TitleRow>
+              <TYPE.largeHeader style={{ justifySelf: 'flex-start' }}>Farm</TYPE.largeHeader>
+            </TitleRow>
+            <Row>
+              <TYPE.darkGray style={{ marginRight: '0.5rem', marginTop: '0.5rem', justifySelf: 'flex-start' }}>
+                Stake your Balancer Pool Tokens (BPT) to earn Halo!
+              </TYPE.darkGray>
+            </Row>
+            <Row>
+              <StyledExternalLink href="https://google.com" style={{ fontSize: '14px' }}>
```
Comment (schystz) [inline src/pages/BalancerPool/index.tsx]: @tracyarciaga maybe we link this to `#` for now until it is available, but definitely avoid google.com :P

File: src/pages/BalancerPool/index.tsx [hunk 3]
```diff
@@ -32,33 +61,58 @@ const BalancerPool = () => {
...
       <PageWrapper>
         <SwapPoolTabs active={'pool'} />
+        <PoolsSummaryRow>
+          <HeaderRow>
+            <TitleRow>
+              <TYPE.largeHeader style={{ justifySelf: 'flex-start' }}>Farm</TYPE.largeHeader>
+            </TitleRow>
+            <Row>
+              <TYPE.darkGray style={{ marginRight: '0.5rem', marginTop: '0.5rem', justifySelf: 'flex-start' }}>
+                Stake your Balancer Pool Tokens (BPT) to earn Halo!
```
Comment (schystz) [inline src/pages/BalancerPool/index.tsx]: @tracyarciaga noticed this line is not exactly the same as figma design

**Review by schystz — COMMENTED** [review]
@tracyarciaga code changes looks good to me in general aside from a few minor comments.

Just pasting here some UI issues Chris and I has observed:

Screen Shot 2021-04-12 at 3 58 16 PM

Screen Shot 2021-04-12 at 3 41 23 PM

Screen Shot 2021-04-12 at 3 41 43 PM

File: public/locales/en.json (hunk 1 above)
Comment (tracyarciaga) [reply public/locales/en.json]: @schystz thanks!

**Comment by tracyarciaga** [comment]
@schystz Pushed update to resolve your comments. Thank you!

**Comment by schystz** [comment]
@tracyarciaga raised my feedback on Slack #design-engineering instead so Jofo can also take a look

**Comment by ernestpascual** [comment]
Hi guys, i raised a PR that might affect this PR https://github.com/HaloDAO/halodao-interface/pull/14 
FYI, thanks!

**Comment by tracyarciaga** [comment]
@schystz Pushed the changes to address your comments:
1.  Added test -> "(Liquidity Pool)" in  the Farm header.
2. Reverted colors for network, balance, address items in navbar.

**Review by schystz — APPROVED** [review]
Looks good on my side, so approving!

**Comment by ernestpascual** [comment]
hi @tracyarciaga  one minor comment:

Screen Shot 2021-04-15 at 1 44 51 AM

I saw this already fix in @schystz previous PR. We can keep it to 2 decimals. Otherwise, im good!

**Comment by ernestpascual** [comment]
on the vesting side i think I have the fix for that, will just merge with this branch after this is merged to develop :)

**Comment by schystz** [comment]
> hi @tracyarciaga one minor comment:
> 
> Screen Shot 2021-04-15 at 1 44 51 AM
> 
> I saw this already fix in @schystz previous PR. We can keep it to 2 decimals. Otherwise, im good!

Actually for this one, we'll be tracking it in a separate task - https://halodao.atlassian.net/browse/HDF-142

**Review by ernestpascual — APPROVED** [review]

**Comment by ernestpascual** [comment]
gotcha @schystz , approved!

**Pull request merged**